nifi-ngsi-bundle/nifi-ngsi-processors/src/main/java/org/apache/nifi/processors/ngsi/ngsi/backends/http/HttpBackend.java

```diff
@@ -226,12 +231,12 @@ public Object run() {
                 Set<Principal> principals = current.getPrincipals();
 
                 for (Principal next : principals) {
-                    System.out.println("DOAS Principal: " + next.getName());
+                    logger.info("DOAS Principal: {}", next.getName());
                 } // for
```

Comment:
something like logger.info(String.join(principals.map("DOAS Principal" + it.getName())))
could repace this logging loop.